parlai/mturk/core/socket_manager.py

```diff
@@ -157,7 +157,7 @@ class SocketManager():
 
     # Time to acknowledge different message types
     ACK_TIME = {Packet.TYPE_ALIVE: 2,
-                Packet.TYPE_MESSAGE: 2}
+                Packet.TYPE_MESSAGE: 0.5}
```

Comment:
I'm assuming changing this to 0.5 vs. the 0.3 in the file above is intentional?

Reply:
Yup - the commands sent by the server have to have a slightly longer delay to be safe, but the javascript doesn't have any command/control from it's end so it can be more aggressive.